irohad/ametsuchi/mutable_storage.hpp

```diff
@@ -50,10 +54,11 @@ namespace iroha {
        * otherwise.
        * @return True if block was successfully applied, false otherwise.
        */
-      virtual bool apply(const model::Block &block,
-                         std::function<bool(const model::Block &,
-                                            WsvQuery &,
-                                            const hash256_t &)> function) = 0;
+      virtual bool apply(
+          const shared_model::interface::Block &block,
```

Comment:
Why was the interface changed in this PR?

Reply:
It was called in chain validator, we are syncing with @x3medima17